daily_milestone.go

```diff
@@ -156,15 +156,33 @@ func getMilestones(baseURL string, token string, projectID string) ([]simpleMile
 }
 
 // CreateMilestoneData creates new milestones with title and due date
-func createMilestoneData(advance int) []simpleMilestone {
+func createMilestoneData(advance int, milestoneTime string) []simpleMilestone {
 	today := time.Now().Local()
 	list := []simpleMilestone{}
 	for i := 0; i < advance; i++ {
-		date := today.AddDate(0, 0, i).Format("2006-01-02")
-		milestone := simpleMilestone{}
-		milestone.Title = date
-		milestone.DueDate = date
-		list = append(list, milestone)
+		if milestoneTime == "daily" {
+			date := today.AddDate(0, 0, i).Format("2006-01-02")
+			milestone := simpleMilestone{}
+			milestone.Title = date
+			milestone.DueDate = date
+			list = append(list, milestone)
+		} else if milestoneTime == "weekly" {
+			date := today.AddDate(0, 0, (i * 7)).Format("2006-01-02")
```

Comment:
What's the reasoning behind i*7? Wouldn't something like using the week specific function work better? https://golang.org/pkg/time/#Time.ISOWeek

Consider the edge case, that weekly, monthly and daily all run on the same repo.

Reply:
For weekly the title and the due date might actually be different. The due date would be the last day of the week, but the title would be something like 2017-W01 or so.

Reply:
Yeah this would be the better option didn't look into it that deeply. Thanks.

Reply:
Yeah this would be the better option didn't look into it that deeply. Thanks.